ATL/schema.sql

```diff
@@ -0,0 +1,53 @@
+-- Passengers 
+ CREATE TABLE Passengers ( 
+	PassengerId INT PRIMARY KEY IDENTITY(1,1) NOT NULL, 
+    FirstName NVARCHAR(50) NOT NULL, 
+    LastName NVARCHAR(50) NOT NULL, 
+    Age INT NOT NULL 
```

Comment:
Storing age like this is a really bad idea. Try to think about why that is, we'll discuss it during the class.